fe/fe-core/src/main/java/org/apache/doris/backup/BackupJobInfo.java

```diff
@@ -204,6 +278,7 @@ public String getOrginNameByAlias(String alias) {
         public String database;
         @SerializedName("backup_time")
         public long backupTime;
+        public BackupContent content;
```

Comment:
No need `@SerializedName`?

Reply:
No, it will be serialized by default, but some serialized names and attribute names are different.